### ISSUE TITLE
Library does not read calculated values

### ISSUE THREAD

**Issue by Youssef93** [michalsnik/aos#579]
## This is:
- Bug

## Specifications
  - AOS version: 2.3.4
  - OS: Windows
  - Browser: Chrome

## Expected Behavior
I'd expect when adding a calculated value for an animation duration or animation delay, it would work.
For example `data-aos-delay="2*500"` this should work. 
This is useful because in my case I am using angular with a for loop and I want each element to be delayed based on its index in the array. `data-aos-delay="i*100"`

## Actual Behavior
I believe in this case the library just goes with the defaulted value.

## Possible Solution
I suggest finding a way to make these fields take a dynamically calculated number

**Comment by AlbertSmit** [comment]
Just to be sure; are you sure that `data-aos-delay="2*500"` would not resolve to be a string in your scenario?

**Comment by Youssef93** [comment]
Yes. I have also tried this `data-aos-delay="parseInt(2*500)"` but did not work either

**Comment by rajivkr** [comment]
I tried `[attr.data-aos-delay]="i * 200"`, seems to be working fine for me.
